perception/lidar_obstacle_detection/centerpoint_backbone/onnx_centerpoint_backbone/definition.yaml

```diff
@@ -49,7 +49,7 @@ network_parameters:
       datatype: float32
       shape:
         - 1
-        - 3
+        - 5
```

Comment:
Was it always supposed to be 5 or is it only with the updated model?

Reply:
The updated model